src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
 import { Header } from "components/Header";
 
+interface AboutLinkProps {
+  url: string;
+  label: string;
+}
+
+const AboutLink: React.FC<AboutLinkProps> = ({ url, label }) => {
```

Comment:
This looks good. You should add `style={{ textDecoration: "none" }}` to the anchor tag (like you had it before 😉) in order to follow the same style as the template picture

```tsx
const AboutLink: React.FC<AboutLinkProps> = ({ url, label }) => {
  return (
    <div>
      <a
        href={url}
        target="_blank"
        rel="noopener noreferrer"
        style={{ textDecoration: "none" }}
      >
        {label}
      </a>
    </div>
  );
};
```